user/views.py

```diff
@@ -82,12 +98,12 @@ def logout(request):
 
 
 def profile(request):
-    """
-    The method views the user profile.
+    """The method views the user profile.
 
     Parameters
     ----------
-    request : to view the user profile.
+    request : request
+        to view the user profile.
```

Comment:
Use complete sentence and describe the parameter itself and not what it is good for.